client/state/data-layer/wpcom-http/index.js

```diff
@@ -60,6 +60,8 @@ export const queueRequest =
 		const { responseType } = options || {};
 		const fetcher = get( options, 'options.fetcher', 'wpcom' );
 
+		const isLocalApiCall = options?.isLocalApiCall;
```

Comment:
code style nit: this can be moved into the `responseType` destructuring call few lines above.